contrib/librespot.service

```diff
@@ -1,5 +1,6 @@
 [Unit]
-Description=Librespot
+Description=An open source Spotify client
```

Comment:
Oh, I'd never looked at the docs for that particular one. On my local system around 3-4 or 401 unit files _actually_ do what the documentation suggests.

Go ahead and run `ag Description /usr/lib/systemd/system`, to see the values of all local units.

I do think that putting the name in the description is weird, since then the name **and** the description contain the same information, which makes the `Description=` field valueless and results in no real description.

If you think that's a blocker I'll change it, but I'd rather follow the de-facto standard, rather than the written standard that is used by <1%.

Reply:
That's not for me to say. I was just pointing out that it doesn't follow the spec.

Reply:
> Go ahead and run `ag Description /usr/lib/systemd/system`, to see the values of all local units.

I did and some follow the spec and some don't. So really it's a tossup. The ones that don't follow the spec prob didn't read the docs either,lol!!!

Reply:
I think it's not too difficult to do both: follow current practice _and_ the spec, if you were to change this into: "Librespot open source Spotify client" or the likes. This way, the `systemd` status messages will make sense _and_ it's clear to the user what `librespot` actually is.